app/lib/menu/menu-builder.js

```diff
@@ -710,7 +718,16 @@ function mapMenuEntryTemplate(entry) {
     label: entry.label,
     accelerator: entry.accelerator,
     enabled: entry.enabled !== undefined ? entry.enabled : true,
-    click: () => app.emit('menu:action', entry.action, entry.options),
+    click: (...args) => {
+      const event = args[2];
+
+      const options = {
+        ...entry.options,
+        triggeredByAccelerator: event.triggeredByAccelerator
```

Comment:
How about if we call it `triggeredByShortcut`? `Accelerator` sounds like Electron domain language.

Reply:
Renamed it accordingly via https://github.com/camunda/camunda-modeler/pull/3255/commits/ee8ce8e56a20a31a93c5bf8c270bacdedfd72150